app/js/PauseButton.js

```diff
@@ -43,11 +43,10 @@ export default class PauseButton extends React.PureComponent<{}, State> {
     );
 
     return (
-      <button
-        className="btn btn-outline-secondary btn-sm"
+      <a
```

Comment:
This should stay a `<button>` since it doesn't link anywhere. Can you style the button the way you want? Bootstrap has a `btn-link` class for this purpose.

Reply:
Will change it.